<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="UP_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DOWN_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RIGHT_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>RIGHT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LEFT_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>LEFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SELECT_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>SELECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>20</width>
      <height>781</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Battery_Label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Battery Level</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="BatteryVar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="Sensor_Label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Sensor Connected </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SensorVar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Attached?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="POWER_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>610</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>POWER</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="PowerBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="Main_Screen">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="achScore">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="timeCounter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="coherenceScore">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="breathPacer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="format">
     <string notr="true">%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Coherence Score</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lowBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mediumBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="highBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="redLight">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueLight">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenLight">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sensorConnect">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensor Connected</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
